### ISSUE TITLE
[NPC] Aku'mai Servant

### ISSUE THREAD

**Issue by PolarCookie** [Atlantiss/NetherwingBugtracker#3892]
**Description**: 
https://youtu.be/Ss0pKe6Q2JY?t=338
There should only be 2 Aku'mai Servants spawned, not 4.

**Server Revision**: 
2602

**Comment by sandfury** [comment]
Also, there should be a lot more of the non-elite crabs spawning than 4, looks like everything is in wave of 4s atm.

**Comment by randomuser69** [comment]
So I was going to make a report that these guys (Aku'mai Servant) are way overtuned. Four of them spawn and they completely demolished both groups both times I ran this instance on separate days. Turns out there's only supposed to be two of them that spawn, as per above report, not four. At the moment, the instance is doable, but very frustrating because these guys do this attack that hits all nearby players (https://classic.wowhead.com/spell=8398/frostbolt-volley) and since theres four of them doing it at once it completely demolishes groups.

@Evolvee 
@wtRCH
